### ISSUE TITLE
monarch run: Cannot find executable for CFBundle

### ISSUE THREAD

**Issue by fogelfish** [Dropsource/monarch#15]
I don't know the cause or effect of **Cannot find executable for CFBundle**, but I thought you should see this.  The sample_button_stories.dart file seems okay in the Monarch app.

```
$ monarch run
Using flutter sdk at /Users/brianfogel/flutter/flutter/bin/flutter
Enabling Flutter for desktop

Starting Monarch

Preparing stories...
3.1sec elapsed, 0/2 tasks completed
6.2sec elapsed, 0/2 tasks completed
9.3sec elapsed, 0/2 tasks completed
12.4sec elapsed, 0/2 tasks completed
15.5sec elapsed, 0/2 tasks completed
18.6sec elapsed, 0/2 tasks completed
21.7sec elapsed, 0/2 tasks completed
24.8sec elapsed, 0/2 tasks completed
27.9sec elapsed, 0/2 tasks completed
31.0sec elapsed, 0/2 tasks completed
34.1sec elapsed, 0/2 tasks completed
37.2sec elapsed, 0/2 tasks completed
40.3sec elapsed, 0/2 tasks completed
43.4sec elapsed, 1/2 tasks completed
46.5sec elapsed, 1/2 tasks completed
49.6sec elapsed, 1/2 tasks completed
52.7sec elapsed, 1/2 tasks completed
Preparing stories completed, took 54.2sec

Launching Monarch app...
2021-04-01 16:20:55.483 monarch[22016:24434604] Cannot find executable for CFBundle 0x7fd3ebe5cdc0 </Users/brianfogel/Sites/playground/.monarch> (not loaded)

3.1sec elapsed, 0/1 tasks completed
Launching Monarch app completed, took 3.4sec

Monarch is ready to use

Setting up stories watch...
3.1sec elapsed, 0/2 tasks completed
6.2sec elapsed, 0/2 tasks completed
9.3sec elapsed, 0/2 tasks completed
12.4sec elapsed, 0/2 tasks completed
15.5sec elapsed, 0/2 tasks completed
18.6sec elapsed, 1/2 tasks completed
21.7sec elapsed, 1/2 tasks completed
24.8sec elapsed, 1/2 tasks completed
27.9sec elapsed, 1/2 tasks completed
31.0sec elapsed, 1/2 tasks completed
Setting up stories watch completed, took 33.0sec
Cannot find executable for CFBundle

```
Flutter version 2.0.3
Dart version 2.12.2

monarch_binaries: 0.3.6
monarch_cli: v0.2.12
monarch_mac_app: v0.1.8

System Version:	macOS 11.2 (20D64)
Kernel Version:	Darwin 20.3.0

**Comment by fertrig** [comment]
Yes, we started seeing that warning recently. It is a warning that only occurs on macOS. It seems related to how we build the Monarch mac app and some setting in Xcode.

It doesn't have any other side effect. Everything in Monarch works as expected.

We are actively working on other high priority issues like #11. I'll leave this issue open so we can track it.

Thanks for documenting this issue!

**Comment by fertrig** [comment]
This issue is fixed with version 1.0.4. After you do `monarch upgrade` you should not see that message anymore. 

[Release notes](https://monarchapp.io/blog).
